File: java/spring-arch-unit/spring-arch-unit-mvc-example/src/main/java/com/bmarques/springarchunitmvcexample/service/InvoiceService.java
package com.bmarques.springarchunitmvcexample.service;

import com.bmarques.springarchunitmvcexample.domain.invoice.Invoice;
import com.bmarques.springarchunitmvcexample.repository.InvoiceRepository;
import org.springframework.stereotype.Service;

@Service
public class InvoiceService {

    private InvoiceRepository repository;

    public Invoice getInvoice() {
        return repository.getInvoice();
    }
}

File: kotlin/spring-arch-unit/spring-arch-unit-mvc-example/src/main/kotlin/com/bmarques/springarchunitmvcexample/service/InvoiceService.kt
package com.bmarques.springarchunitmvcexample.service

import com.bmarques.springarchunitmvcexample.domain.invoice.ElectronicInvoice
import com.bmarques.springarchunitmvcexample.repository.InvoiceRepository
import org.springframework.stereotype.Service

@Service
class InvoiceService(private val repository: InvoiceRepository) {

    fun getInvoice(): ElectronicInvoice = repository.getInvoice()
}